.github/workflows/deploy-prod-digdir.yaml

```diff
@@ -1,71 +0,0 @@
-name: Build, test and deploy to production (digdir) on merge to master branch
```

Comment:
Har fjernet støtte for lesebrukere, så avventer deploy til prod og demo til sso er klar